### PR TITLE
Update changlog for 1.2.2

### PR DESCRIPTION
**Pull request by bhennion**
@xournalpp/core please check in case I forgot something.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+* Fix reference handling of sidebar widgets, fixing crashes and warning (#5083,
+  #5074, #4026)
+* Default build type is now "RelWithDebInfo" (#5118)
+* Fix error message when trying to open an autosave file (#5140)
+* Fix empty toolbars being shown (#5160)
+* Add a device id check for input sequences, so devices don't interfere with one
+  another anymore (#5170, #5116, #5087)
+* Fix a bunch of memory leaks (#5164)
+* Properly read PDF files as binary, fixing some reading issues (#5174, #4740)
+* Remove unsafe symbols from document name generation (#5144, #5126)
```
Comment (rolandlo) [inline CHANGELOG.md:17]:
```suggestion
* Remove unsafe symbols from document name generation (#5144, #5123)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
+* Fix crash when a modification happens outside of the cached pages (#5144, 
+  #5178, #5162, #5046)
```
Comment (rolandlo) [inline CHANGELOG.md:19]:
```suggestion
  #5178, #5162, #5046, #5126)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,23 @@
...
 ## 1.2.2~dev (Unreleased)
 
+Bugfixes from various community contributors.
+
+* Fix undefined behaviour due to signal callbacks wrong type (#5057)
+* Fix reference handling of sidebar widgets, fixing crashes and warning (#5083,
+  #5074, #4026)
+* Default build type is now "RelWithDebInfo" (#5118)
+* Fix error message when trying to open an autosave file (#5140)
+* Fix empty toolbars being shown (#5160)
```
Comment (rolandlo) [inline CHANGELOG.md:12]:
```suggestion
* Fix empty toolbars being shown (#5160, #5051)
```

**Review by rolandlo — CHANGES_REQUESTED** [review]
Maybe we should also add something like:
"To prevent this popup from reappearing, simply delete the errorlogfiles." to
https://github.com/xournalpp/xournalpp/blob/5fd2dd3703ef7b7cd07e63ef738fc0affcaa287f/src/core/control/XournalMain.cpp#L184-L188

This is unclear to some users.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,23 @@
...
+* Add a device id check for input sequences, so devices don't interfere with one
+  another anymore (#5170, #5116, #5087)
+* Fix a bunch of memory leaks (#5164)
```
Comment (bhennion) [inline CHANGELOG.md:15]:
```suggestion
* Fix a bunch of memory leaks (#5164, #5216)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,23 @@
...
+* Fix error message when trying to open an autosave file (#5140)
+* Fix empty toolbars being shown (#5160, #5051)
+* Add a device id check for input sequences, so devices don't interfere with one
+  another anymore (#5170, #5116, #5087)
+* Fix a bunch of memory leaks (#5164, #5216)
+* Properly read PDF files as binary, fixing some reading issues (#5174, #4740)
+* Remove unsafe symbols from document name generation (#5144, #5123)
+* Fix crash when a modification happens outside of the cached pages (#5144, 
+  #5178, #5162, #5046, #5126)
+* Fix audio output settings being off by 1 (#5205)
```
Comment (bhennion) [inline CHANGELOG.md:20]:
```suggestion
* Fix audio output settings being off by 1 (#5205)
* Clarify CrashLog message: deleting the crashlog will stop the message from popping up
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,24 @@
...
+* Clarify CrashLog message: deleting the crashlog will stop the message from popping up (#5214)
```
Comment (bhennion) [inline CHANGELOG.md:21]:
```suggestion
* Clarify CrashLog message: deleting the crashlog will stop the message from popping up (#5214)
* Fix a crash when pressing both mouse buttons at once (#5210, #5149)
```

**Pull request closed without merging**